tests/filter/file.t

```diff
@@ -43,7 +43,7 @@
   * add file2
   * add file1
 
-  $ josh-filter -s --squash --file file.josh
+  $ josh-filter -s --single --file file.josh
```

Comment:
now that I think about it, it's weird that the result of this is the "initial" commit and not one of the three above...